test/integration/configuration/test_profile_jinja.py

```diff
@@ -246,7 +246,6 @@ def test_profile_detect_os_arch(self):
         the_os = pr.settings['os']
         arch = pr.settings['arch']
         expected = textwrap.dedent(f"""\
-            Host profile:
```

Comment:
`out` reorders this output enough for this line to be missplaced in tests, which is now output to stderr, so no functional change here